packages/ia-topnav/src/desktop-subnav.js

```diff
@@ -0,0 +1,44 @@
+import { html } from 'lit-element';
+import { more } from './data/menus';
+import TrackedElement from './tracked-element';
+import desktopSubnavCSS from './styles/desktop-subnav';
+import icons from './assets/img/icons';
+
+class DesktopSubnav extends TrackedElement {
+  static get styles() {
+    return desktopSubnavCSS;
+  }
+
+  static get properties() {
+    return {
+      baseUrl: { type: String }
+    };
+  }
+
+  get listItems() {
+    return more(this.baseUrl).map(link => (
+      html`
+        <li>
+          <a class="${link.label.toLowerCase()}" href="${link.url}">${link.label}${DesktopSubnav.iconFor(link.label)}</a>
+        </li>
+      `
+    ));
+  }
+
+  static iconFor(label) {
+    const subnavIcons = {
+      Donate: icons.donate
+    };
+    return subnavIcons[label] ? subnavIcons[label] : html``;
```

Comment:
How come you construct that `subnavIcons` object, then pull the label from that? Can you reference them directly from `icons.donate`?

Reply:
There was previously (and possibly added again in the future) an upload link that was a standalone link within the subnav, so this made it easier to determine which to render.